app.go

```diff
@@ -584,30 +590,33 @@ func (app *App) SetTLSHandler(tlsHandler *TLSHandler) {
 // any of the fiber's sub apps are added to the application's error handlers
 // to be invoked on errors that happen within the prefix route.
 func (app *App) Mount(prefix string, fiber *App) Router {
-	stack := fiber.Stack()
 	prefix = strings.TrimRight(prefix, "/")
 	if prefix == "" {
 		prefix = "/"
 	}
 
-	for m := range stack {
-		for r := range stack[m] {
-			route := app.copyRoute(stack[m][r])
-			app.addRoute(route.Method, app.addPrefixToRoute(prefix, route))
-		}
-	}
-
 	// Support for configs of mounted-apps and sub-mounted-apps
 	for mountedPrefixes, subApp := range fiber.appList {
 		app.appList[prefix+mountedPrefixes] = subApp
-		subApp.init()
```

Comment:
who is initalizing the app now ?
https://github.com/gofiber/fiber/blob/ce2d087394dc74dbe45e6c9f9bb263ae6f105a26/app.go#L952

this is necessary so that the render features also work together with the subapp
https://github.com/gofiber/fiber/blob/ce2d087394dc74dbe45e6c9f9bb263ae6f105a26/ctx.go#L1401

but maybe we should do this later in the init function of the parent app

if this has not been noticed by then, it seems as if we still need a test for this case. -> pls add a testcase for this mount app case from #1749